contributing/code/standards.rst

```diff
@@ -175,6 +175,9 @@ Structure
   switched to opt-in via ``@`` operator.
   Read more at :ref:`contributing-code-conventions-deprecations`.
 
+* Do not use ``else``, ``elseif``, ``break`` after ``if`` and ``case`` conditions
+  which returns or throws something.
```

Comment:
I'm not sure, but should we say `... which return or throw something` instead of `... which returnS or throwS something` ?